addons/hr_holidays/models/hr_holidays.py

```diff
@@ -442,7 +454,7 @@ def action_approve(self):
         return True
 
     @api.multi
-    def _prepare_create_by_category(self, employee):
+    def _prepare_create_by_batch(self, employee):
```

Comment:
The new name is not clearer than the old one ...

Reply:
/agree